src/pug/blog/report_25.pug

```diff
@@ -0,0 +1,98 @@
+
+  p The modified v7 prototype has substantially better antenna performance, but it is still far from reaching its full potential, reason being that with the "trim tuning" method we can only tune the antenna itself but not the inductance/capacitance of the matching network. To actually tune the matching network we would need more advanced tools and expertize we don't have, or outsource the tuning to an external company (expensive).
+
+  p We also found that by reducing the rate of the #[em ESB] (Enhanced Shock Burst) connection from #[em 2 Mbps] to #[em 250 Kbps] (8x times slower) the reception sensitivity increases by several decibels. #[i (see table below)]
+  ul
+    li At 2 Mbps the range is quite short and there are micro-cuts while changing the antenna orientation.
+    li At 250 Kbps the range is much better and its much more stable while changing orientation.
+
+  p This is an acceptable tradeoff with a negligible impact on latency (~400 microseconds), but has some #[em negative effects]: More chances of interference in congested WLAN/BT environments, potencially slower transference of data from/to the Ctrl app, and not being able to use 2 controllers at once (for example in couch coop games).
+
+  p Since these results are not as good as we expected, we decided to experiment with #[em Espressif ESP] all-in-one modules as secondary communication unit, with the goal of replacing NordicSemi NRF24 and antenna with it, while keeping the Raspberry RP2040 as the primary microcontroller.
+
+  p We evaluated #[em ESP C-series] (RISC-V) Mini modules, using the #[em ESP-NOW] protocol, and measuring latency, jitter, range and latency in the same way we did with our prototypes.
+
+  +figure(
+    '',
+    'ESP-C6-MINI test',
+    '/static/blog/ESP-C6-MINI.jpg',
+    'small inline',
+  )
+  +figure(
+    '',
+    'Idea for next prototype',
+    '/static/blog/RF_proto_8_render.jpg',
+    'small inline',
+  )
+
+  table.inline.mono
+    tr
+      th
+      th Proto v7 (ESB 2 Mbps)
+      th Proto v7 (ESB 250 Kbps)
+      th ESP32-C6-MINI (ESP-NOW)
+    tr
+      th Latency (99%)
+      td < 1 ms
+      td < 1 ms
+      td 1.4 ms
+    tr
+      th Jitter (1%)
+      td +1 ms
+      td +1 ms
+      td +2 ms
+    tr
+      th Usable range
+      td ~ 4 meters
+      td ~ 10 meters
+      td ~ 25 meters
+    tr
+      th Stability
+      td Very bad
+      td Acceptable
+      td Excellent
+
+  p Moving forward we are planning to test current v7 #[em RP2040+Nordic] prototypes, and upcoming v8 #[em RP2040+ESP] prototypes end-to-end on real controllers in real environments, and gather feedback from a few selected testers.
+
+  hr
+  h3 Finland VAT increase
+  p Finland Value-Added Tax has increased from #[em 24%] to #[em 25.5%] after september 2024, and we updated the shop accordingly.
+
+  p We decided not to increase the final price of the parts, so technically the parts are cheaper now, but in practice there is no change for the people using the shop.
```

Comment:
The parts are not cheaper, but the same price (to the end user). IL is continuing to make less profit per unit.